tests/admin_views/tests.py

```diff
@@ -6206,6 +6206,22 @@ def test_first_field_focus(self):
         )
         self.take_screenshot("focus-multi-widget")
 
+    @screenshot_cases(["desktop_size", "mobile_size", "rtl", "dark", "high_contrast"])
+    def test_error_message_position(self):
+        """Error messages are shown above the field and below the label"""
```

Comment:
```suggestion
    def test_error_message(self):
```
As we're not asserting based off the position (it won't fail if we move it) - I would call this something generic on errors